src/cluster.c

```diff
@@ -107,6 +108,33 @@ ConnectionType *connTypeOfCluster(void) {
     return connectionTypeTcp();
 }
 
+void delUnOwnedKeys(void) {
+    if (!server.cluster_enabled) {
+        return;
+    }
+    unsigned int deleted_keys = 0;
+    unsigned int unowned_slots = 0;
+    clusterNode *master;
```

Comment:
Recently we have moved to `primary/replica` terminology.

```suggestion
    clusterNode *primary;
```